reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-60.md

```diff
@@ -0,0 +1,345 @@
+# What's New in PowerShell Core 6.0
+
+## Renamed `powershell(.exe)` to `pwsh(.exe)`
+
+The binary name for PowerShell Core has been changed from `powershell(.exe)` to `pwsh(.exe)` in order to provide a deterministic way for users to open PowerShell Core on machines where both Windows PowerShell and PowerShell Core are installed.
+
+`pwsh` is also much shorter and easier to type.
+
+Additionally, we've made some changes to `pwsh(.exe)`:
+
+- Change positional parameter for `powershell.exe` from `-Command` to `-File`.
+ This fixes the usage of `#!` (aka as a shebang) in PowerShell scripts that are being executed from non-PowerShell shells on non-Windows platforms.
+ This also means that you can now do things like `powershell foo.ps1` or `powershell fooScript` without specifying `-File`.
+ However, this change now requires that you explicitly specify `-c` or `-Command` when trying to do things like `powershell.exe Get-Command`.
+ (#4019)
+- Update PowerShell Core to accept the `-i` (or `-Interactive`) switch to indicate an interactive shell. (#3558)
+ This will help when using PowerShell as a default shell on Unix platforms.
+- Remove parameters `-importsystemmodules` and `-psconsoleFile` from `pwsh.exe`. (#4995)
+- Fix `pwsh -version` and built-in help for `pwsh.exe` to align with other native tools. (#4958 & #4931) (Thanks @iSazonov)
+- Make invalid argument error messages for `-File` and `-Command` consistent and make exit codes consistent with Unix standards (#4573)
+- Add `-WindowStyle` parameter on Windows. (#4573)
+
+## Moved from .NET Framework to .NET Core
+
+TODO
+
+## Support for macOS and Linux
```

Comment:
This should be much higher up.  Probably right after `Windows PowerShell and PowerShell Core 6 side-by-side`